packages/manager/modules/hub/src/components/product-list/config/domain.config.js

```diff
@@ -0,0 +1,50 @@
+export default {
+  route: '/domain/{domain}',
+  columns: [
```

Comment:
Looks like transclusion with extra steps

```js
 <ovh-manager-hub-user-product-tile how-to-go-to-product-page="$ctrl.goToDomain">
    <ovh-manager-hub-user-product-tile-column>
        <span data-translate="'hub_product_listing_offer_' + $row.offer"></span>
    </ovh-manager-hub-user-product-tile-column>

...and so on
</ovh-manager-hub-user-product-tile>
```

Reply:
no, those are the columns passed to the oui-datagrid 'columns' attribute ; this is how dynamic columns are handled in the oui-datagrid